game.py

```diff
@@ -38,7 +39,9 @@
 
 ply = {
     "energy": 24,
-    "hp": 100
+    "max_energy": 24,
```

Comment:
good thinking ahead on these. you can tell i was thinking about min max values with the eventual thoughts of having buffs from items or purchased abilities or something.